.github/workflows/ci.yml

```diff
@@ -113,80 +132,21 @@ jobs:
   # a single report, we have to tell Coveralls when we've uploaded all of the
   # data files.  This does it.  We make sure it runs last by making it depend
   # on *all* of the coverage-collecting jobs.
+  #
+  # See notes about parallel builds on GitHub Actions at
+  # https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html
   finish-coverage-report:
-    # There happens to just be one coverage-collecting job at the moment.  If
-    # the coverage reports are broken and someone added more
-    # coverage-collecting jobs to this workflow but didn't update this, that's
-    # why.
-    needs:
-      - "coverage"
-    runs-on: "ubuntu-latest"
+    needs: coverage
```

Comment:
Can you make sure all yaml strings being added here are quoted?

Also, it seems like the `needs` list may as well stay as a list.  If we end up adding another coverage-collecting job it will need to become a list again anyway.